### ISSUE TITLE
npcap 0.86 stopped working with the newest windows 10 creators edition

### ISSUE THREAD

**Issue by zdm** [nmap/nmap#891]
It just not installing, possible certificate error again?

**Comment by zdm** [comment]
![screenshot 2017-05-26 21 42 12](https://cloud.githubusercontent.com/assets/1025608/26508200/79aaeeba-425c-11e7-89c6-59759906a942.png)

**Comment by hsluoyz** [comment]
@zdm , please try the latest release.

**Comment by taismi** [comment]
I use 0.90 and it works with the Creators update. 

What Wi-Fi device you are using? Does wireless toolbar work in Wireshark?

I have Netgear (Broadcom BCM43526) and toolbar doesn't work. 

Seems that 802.11ac/VHT is not suported. Ht is supported but I cannot set channel 36+40.
C:\Windows\System32\Npcap>WlanHelper.exe 8c309594-364d-425a-98f7-7ba157b84aa6 modus
ht, ofdm, erp, hrdsss

**Comment by zdm** [comment]
v0.86 works perfectly three days ago.
I updated windows to creators edition - and installed npcap stopped working.
Then I tried to reinstall npcap 0.86, and 0.90 - and they are throw error during installation.

**Comment by hsluoyz** [comment]
Hi @taismi , please open another issue if it's not the same question, thanks!

**Comment by hsluoyz** [comment]
OK. So the issue is the system restore point setting fails with error 1058. Please turn on the System Protection on your Windows:

1. Press `Win+R` to open a `Run` window.
2. Type in `control sysdm.cpl,,4` and press `Enter`.
3. Click ``Configure...``, click ``Turn on system protection``.

**Comment by w1nst0nwU** [comment]
Hi,all.
i installed npcap 0.90 successfully,but it still doesn't work on the newest windows 10 creators edition @zdm @hsluoyz

**Comment by hsluoyz** [comment]
Hi @w1nst0nwU , if your error is not about ``system restore 1058``, please open another issue for your problem, thanks!

**Comment by zdm** [comment]
So, I enabled restore points - this was not help.
Then I manually found and removed all wpcap.dll from system drive.
Only after this I can install npcap v0.90.
Now it works for me.

Thoughts:
1. It should installs without restore points enabled;
2. Installation process is not very reliable in some cases;

Thank you, you can close this issue.

**Comment by hsluoyz** [comment]
Hi @zdm ,

> It should installs without restore points enabled;

As the mechanism of Windows, the OS will create restore point for every kernel driver. And currently, Npcap installer just disables the default restore point and create on its own (the benefit is we can customize the restore point name). So even if Npcap installer disables setting restore points manually, OS will still create one.

> Installation process is not very reliable in some cases;

Yes, sometimes not reliable, because OS status and installed drivers vary. we have to solve such problems case by case.
